### ISSUE TITLE
Doesn't work with networkx 1.8.1

### ISSUE THREAD

**Issue by slavkoja** [christiangoltz/shaape#12]
Hi, i tried it with Debian stable (networkx 1.7~rc1), and all seems to work, then i tried some examples from README on Debian testing (networkx 1.8.1) and i get this error:

```
/usr/local/lib/python2.7/dist-packages/shaape/overlayparser.py:188: RuntimeWarning: The diagram contains objects. that have an ambiguous z-order. Shaape estimates their z-order.
  warnings.warn("The diagram contains objects. that have an ambiguous z-order. Shaape estimates their z-order.", RuntimeWarning)
Traceback (most recent call last):
  File "/usr/local/bin/shaape", line 6, in <module>
    shaape.run.run()
  File "/usr/local/lib/python2.7/dist-packages/shaape/run.py", line 123, in run
    shaape.run()
  File "/usr/local/lib/python2.7/dist-packages/shaape/run.py", line 77, in run
    parser.run(raw_data, objects)
  File "/usr/local/lib/python2.7/dist-packages/shaape/overlayparser.py", line 189, in run
    for cycle in cycles:
  File "/usr/lib/pymodules/python2.7/networkx/algorithms/cycles.py", line 204, in simple_cycles
    for nbr in G[thisnode]:
  File "/usr/lib/pymodules/python2.7/networkx/classes/graph.py", line 319, in __getitem__
    return self.adj[n]
KeyError: <shaape.polygon.Polygon object at 0x7fd8b609b3d0>
```

This error happens with more examples from README and this one comes from:

```
 +-----+
 | abc |
 +-----+
```

My knowledge is not enought to decide if it is a networkx or shaape issue.

regards

**Comment by christiangoltz** [comment]
Thanks for the report slavkoja, do you know which version of shaape you are using?
I can remember that I fixed something similar on the master branch already. If you didnt try yet, you can check if it works with the latest shaape code from the master branch on github. I am travelling without a computer at the moment and will have a chance to check the issue in about 3 weeks.

**Comment by slavkoja** [comment]
I am sorry, i forgot to mention the version - it is a 1.0.0 installed by pip. I manualy install the networkx 1.7 (no other modele changes), and with this version all works. I can try the master branch in the next days.

**Comment by slavkoja** [comment]
I can confirm, that you are right and the master branch works with networkx 1.8.1 :-)

Any plans to release new version, please?

**Comment by christiangoltz** [comment]
Yes I will do so as soon as I am back ;)

**Comment by christiangoltz** [comment]
Okay finally, changes have been released, including this fix as version 1.1.0.
